cibuildwheel/bashlex_eval.py

```diff
@@ -85,9 +85,7 @@ def evaluate_nodes_as_compound_command(nodes: Sequence[bashlex.ast.node], contex
         if node.kind == 'command':
             result += evaluate_command_node(node, context=context)
         elif node.kind == 'operator':
-            if node.op == ';':
-                pass
-            else:
+            if node.op not in {';'}:
```

Comment:
Or `node.op != ';'`, for now? I don't know if/how this would expand

Reply:
```suggestion
            if node.op != ';':
```

I was thinking that if another op came up, it would be passed too, but maybe only `;` should ever pass.

Reply:
Yes, you're right, but I think this has been reasonably stable for a long time, so I reckoned this was slightly premature (and not just a rewrite?). I'm fine with keeping this as well, btw.

Reply:
henry's suggestion seems good to me. i guess maybe i was thinking that other operations would have other behaviours, but I guess that never happened :)